README.md

```diff
@@ -0,0 +1,98 @@
+# Joy2Twist
+
+Dockerized ROS node allowing to control ROS powered mobile robots with Logitech F710 gamepad. Joy2Twist node is converting `sensor_msgs/Joy` message to `geometry_msgs/Twist` in order to provide velocity commands for mobile robot.
+
+## Setup joy
+
+Connect joy via nano USB receiver and make sure it is in **DirectInput Mode** (switch in front o the pad with letters **D** and **X**, select **D**).
+
+To test if joy works use `jstest /dev/input/js0`.
+If the output is:
+
+```
+jstest: No such file or directory
+```
+
+See `ls /dev/input | grep js` and find your joy number. If it differs apply changes in *compose.*.yaml* and launch file.
+
+## Button mapping
+
+|  Button  |      Function      |
+|:--------:|:------------------:|
+|   `LB`   |   enable driving   |
+|   `RB`   | slow driving mode  |
+|   `RT`   |  fast driving mode |
+
+If neither `RB` nor `RT` are pressed robot operates in *normal* driving mode.
```

Comment:
It should be "[...] pressed, the robot [...]".

Reply:
:heavy_check_mark: